lib/course_planner_web/controllers/setting_controller.ex

```diff
@@ -63,7 +63,7 @@ defmodule CoursePlannerWeb.SettingController do
           changesets
           |> Enum.sort_by(&(Changeset.get_field(&1, :key)), &<=/2)
           |> Settings.wrap()
-          |> Map.put(:action, :update)
```

Comment:
this action is not related, this is an [Ecto.Changeset action](https://hexdocs.pm/ecto/Ecto.Changeset.html#module-changeset-actions)